cmd/queue/main.go

```diff
@@ -176,10 +185,14 @@ func knativeProxyHeader(r *http.Request) string {
 // Make handler a closure for testing.
 func handler(reqChan chan queue.ReqEvent, breaker *queue.Breaker, handler http.Handler, prober func() bool) func(http.ResponseWriter, *http.Request) {
 	return func(w http.ResponseWriter, r *http.Request) {
+		reqCtx, probeSpan := trace.StartSpan(r.Context(), "probe")
 		ph := knativeProbeHeader(r)
 		switch {
 		case ph != "":
 			if ph != queue.Name {
+				probeSpan.Annotate([]trace.Attribute{
+					trace.StringAttribute("queueproxy.probe.error", fmt.Sprintf(badProbeTemplate, ph))}, "error")
+				probeSpan.End()
```

Comment:
should it be under `http.Error` ? And same below?

Reply:
That's fine -- just so the `http.Error()` is also included within the span time or is there another reason?

Reply:
well that's the work done in scope of the span, so I think it should be accounted for.